aws/rust-runtime/aws-credential-types/src/cache/lazy_caching.rs

```diff
@@ -228,6 +240,38 @@ mod builder {
             self
         }
 
+        /// A random percentage by which buffer time is jittered for randomization.
+        ///
+        /// For example, if credentials are expiring in 15 minutes, the buffer time is 10 seconds,
```

Comment:
It seems like the Go V2 jitter applies to the whole expiration window rather than to the buffer time. What is your reasoning for applying it to the 10 second buffer time?

Reply:
Go V2's term [ExpiryWindow](https://github.com/aws/aws-sdk-go-v2/blob/1c05fb6452a1f74985ff6deb7a642b9eb441274a/aws/credential_cache.go#L16-L23) actually means the buffer time in the Rust SDK.
```
ExpiryWindow will allow the credentials to trigger refreshing prior to the credentials actually expiring.
```
In our case, `ExpiringCache` automatically takes care of buffer time, specifically within the [expired](https://github.com/awslabs/smithy-rs/blob/d040f76fd1ae0295b9f4475d994e3dd4f4aa8893/aws/rust-runtime/aws-credential-types/src/cache/expiring_cache.rs#L101-L103) function.